release-tools/prow.sh

```diff
@@ -200,9 +206,10 @@ configvar CSI_PROW_HOSTPATH_CANARY "" "hostpath image"
 # all generated files are present.
 #
 # CSI_PROW_E2E_REPO=none disables E2E testing.
-configvar CSI_PROW_E2E_VERSION_1_13 v1.14.0 "E2E version for Kubernetes 1.13.x" # we can't use the one from 1.13.x because it didn't have --storage.testdriver
+# TOOO: remove versioned variables and make e2e version match k8s version
```

Comment:
s/TOOO/TODO?

Reply:
True, but that can't be fixed here. It went into csi-release-tools in https://github.com/kubernetes-csi/csi-release-tools/commit/e0fde8c4f5684f5c259c8c7a1d116def587cabf0 and would have to be fixed there.

Not worth it, right? :grimacing: